### DISCUSSION TITLE
Large enclosure mounting

### DISCUSSION THREAD

**Discussion by brisketfat** [hatlabs/discussions#46] [Q&A] [answered]
I'm sorry for the ridiculous question, but I'm not understanding how to mount a Raspberry Pi and a SH-ESP32 in the large enclosure with the included mounts. Is there a picture floating around that illustrates what I'm missing?

**Comment by mairas** [comment, marked as answer]
Not a ridiculous question at all! Or - wait - SH-ESP32? I guess there would be totally reasonable use cases to have that in the same enclosure as a Raspberry Pi. Is that what you meant?

I wrote the text below assuming an SH-RPi: See the photo below of an older installation. I removed the Pi and the SH-RPi v1 to show the mounts.

The short hex standoffs are first attached to the plastic mounts with nuts underneath, and then come the Pi and the SH-RPi. The fan position might need to be adjusted for SH-RPi v2.

![PXL_20230317_080202730](https://user-images.githubusercontent.com/407010/225847730-fabebdd7-c8c2-4e3f-9a76-0eb14e09a0b2.jpg)

**Reply by brisketfat** [reply]
Yes I must have misunderstood. Thank you for the picture, this makes much more sense now!
